recipes/napari-annotatorj/yum_requirements.txt

```diff
@@ -0,0 +1,6 @@
+mesa-libGL-devel
+xorg-x11-server-Xvfb
+libXrender
+mesa-dri-drivers
+dbus-libs
+libxkbcommon-x11
```

Comment:
Are you sure that all of these are required? If you want to fix the import error, only `mesa-libGL` should be required: https://conda-forge.org/docs/maintainer/maintainer_faq.html#mfaq-libgl-so-1

Reply:
on this I relied on the suggested change by @jaimergp

Reply:
I've seen X errors coming out of nowhere later in the feedstocks, so at this point I just add these yum reqs every time. Different parts of the X stack are needed depending on which parts of napari are imported by the plugin...

Reply:
Ok, then I will add the yum_requirements again